src/actions/index.js

```diff
@@ -1,59 +1,55 @@
 import 'whatwg-fetch';
+import { createAction } from 'redux-actions';
+import {
+  REQUEST_WEEKNUMBER,
+  RECEIVE_WEEKNUMBER,
+  REQUEST_BUGSDIFF,
+  RECEIVE_BUGSDIFF,
+  RECEIVE_BUGSHISTORY,
+  REQUEST_BUGSHISTORY,
+} from '../constants/actionTypes';
+
+const fetchAPI = url => fetch(`/api/${url}`).then(response => response.json());
```

Comment:
good one! will this work in most use cases of using the fetch API? (like if we need to change the http verb for example or pass additional http headers/parameters)